sharktank/sharktank/layers/conv.py

```diff
@@ -58,9 +58,9 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
             x = ops.elementwise(torch.mul, x, self.premul_input)
 
         if q_input is not None:
-            x = q_input.quantize(x)
+            x = ops.quantize(x, q_input)
```

Comment:
You shouldn't need to make any changes to the `layers/` directory, Quantizer.quantize should redirect to ops.quantize, and I think that looks nicer too.